modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/text/TextDDMFormFieldTypeSettingsTest.java

```diff
@@ -153,6 +153,13 @@ public void testCreateTextDDMFormFieldTypeSettingsDDMForm() {
 		Assert.assertEquals("radio", indexTypeDDMFormField.getType());
 		Assert.assertTrue(indexTypeDDMFormFieldOptionsValue.contains("text"));
 
+		DDMFormField hideFieldDDMFormField = ddmFormFieldsMap.get("hideField");
+
+		Assert.assertNotNull(hideFieldDDMFormField);
+		Assert.assertNotNull(hideFieldDDMFormField.getLabel());
+		Assert.assertEquals(
+			"true", hideFieldDDMFormField.getProperty("showAsSwitcher"));
+
```

Comment:
@brunofariasdeo I think this should be before the `indexTypeDDMFormField ` (line 143).